roles/docker/tasks/install.yml

```diff
@@ -69,6 +85,26 @@
   when: inventory_hostname in groups['gnbsim_nodes']
   become: true
 
+- name: Create docker group
+  group:
+    name: docker
+    state: present
+  when: inventory_hostname in groups['gnbsim_nodes']
+  become: true
+
+- name: Add current user to docker group
+  user:
+    name: "{{ ansible_user }}"
+    groups: docker
+    append: yes
+  when: inventory_hostname in groups['gnbsim_nodes']
+  become: true
```

Comment:
Is not it `root` the current user? Is this the intention? or is it the intention add the `actual` user to the Docker group?

Reply:
The intention is to add the current user to the Docker group:
`sudo usermod -aG docker $USER`